rotkehlchen/tests/api/test_sushiswap.py

```diff
@@ -48,25 +39,28 @@ def test_get_balances_module_not_activated(
     )
 
 
-@pytest.mark.parametrize('ethereum_accounts', [[SWAP_ADDRESS]])
+@pytest.mark.parametrize('ethereum_accounts', [['0x31089Ef6f99FB83F95178DBBf08A7A4Bf2eC9fd2']])
```

Comment:
Shouldn't this be VCRed? Or we want it to specifically query online nodes? If we do, why do we do that?